src/tests/ftest/pool/create_all_vm.py

```diff
@@ -72,13 +77,55 @@ def test_one_pool(self):
                 "Invalid SCM size: want={}, got={}, delta={}".format(self.scm_avail_bytes,
                     tier_bytes[0], self.delta_bytes))
         self.assertEqual(0, tier_bytes[1],
-                "Invalid SMD size: want=0, got={tier_bytes[1]}")
+                "Invalid SMD size: want=0, got={}".format(tier_bytes[1]))
 
         self.log.info("Checking size of available storage")
-        self.scm_avail_bytes = self.get_pool_available_bytes()
+        self.scm_avail_bytes = self.get_available_bits()
         self.assertEqual(0, self.scm_avail_bytes,
                 "Invalid SCM size: want=0, got={}".format(self.scm_avail_bytes))
 
+    def test_rank_filter(self):
+        """Test the creation of one pool with filtering the rank to use
+
+        Test Description:
+            Create a pool with all the capacity of some servers. Verify that the pool created
+            effectively used all the available storage of the selected servers and the non selected
+            servers keep their available storage.
+
+        :avocado: tags=all,pr
+        :avocado: tags=vm
+        :avocado: tags=pool,pool_create_all
+        :avocado: tags=pool_create_all_rank_filter_vm,test_pool_rank_filter
```

Comment:
Minor - `test_` tag should match the test method name
```suggestion
        :avocado: tags=pool_create_all_rank_filter_vm,test_rank_filter
```

Reply:
- [ ] fix misnamed tag